gems/sorbet-static-and-runtime/sorbet-static-and-runtime.gemspec

```
@@ -0,0 +1,18 @@
+Gem::Specification.new do |s|
+  s.name        = 'sorbet-static-and-runtime'
+  s.version     = '0.0.0'
+  s.summary     = 'A Typechecker for Ruby'
+  s.description = 'Sorbet static and runtime in one gem'
+  s.authors     = ['Stripe']
+  s.email       = 'sorbet@stripe.com'
+  s.homepage    = 'https://sorbet.run'
```

```suggestion
  s.homepage    = 'https://sorbet.org'
```

It's possible that the other gems say `sorbet.run` but that's probably an artifact of a time when we didn't have `sorbet.org` yet.